thirdeye/thirdeye-frontend/app/pods/rootcause-entities-cache/service.js

```diff
@@ -12,7 +12,7 @@ export default Ember.Service.extend({
 
   pending: null, // Set
 
-  errors: null, // Set({ frameowrk, error })
+  errors: null, // Set({ framework, error })
```

Comment:
nit: let's document using the JS doc format. http://usejsdoc.org/
This should be straightforward. @ttbach  and I are available to discuss this as well.

Reply:
👍. I think as we do more work on this, we should start being more diligent about documentation, so it's easier to read and also to refactor later. It would be unideal if we got to a size that's unmanageable, and we have little clue what's going on just from glancing at it.

Reply:
Yeah I agree it's time to start documenting this. The RCA core UI has become fairly stable.